app/channels/publish_channel.rb

```diff
@@ -34,23 +34,30 @@ def receive(data)
 	protected
 
   def validate_publisher_channel_id_format
-    if !@publisher_channel_id
-      transmit(error: "publisher channel is missing")
+    if @publisher_channel_id.blank?
+      Rails.logger.info("transmit block here")
+      transmit(format_error("Publisher Channel Missing"))
+      false
+    elsif @publisher_channel_id =~ /...../
```

Comment:
am I reading this right that you are limiting publisher channel ids to 5 characters?

Reply:
This regex is not anchored; it merely tests length >= 5. The string length function would be a better way to do this.

Reply:
thanks mark, I completely missed the lack of anchors :)

Reply:
Yeah -- also just want to note that I intended this as more of a stub than anything.  We can tweak that regex to whatever format you want to enforce.

Reply:
@frett what restrictions did you have in mind? Something like alphanumeric and '-' characters only, with a minimum length, would be a superset of the UUID pattern. Could go case insensitive as well, as I believe UUIDs are.

Reply:
My suggestion: `elsif @publisher_channel_id =~ /\A[a-zA-Z0-9-]{5,200}\z/`